charts/mailu/README.md

```diff
@@ -217,7 +218,7 @@ Check that the deployed pods are all running.
 | Name                                                | Description                                                                                                                                                                                               | Value                             |
 | --------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | --------------------------------- |
 | `externalDatabase.enabled`                          | Set to true to use an external database                                                                                                                                                                   | `false`                           |
-| `externalDatabase.type`                             | Type of the external database for mailu and roundcube (`mysql`/`postgresql`)                                                                                                                              | `""`                              |
```

Comment:
Hi @ember-ana , these are generated from the values.yaml, can you rebase an put these changes into the values.yaml as well?